lib/suspenders/app_builder.rb

```diff
@@ -231,6 +231,10 @@ def create_shared_javascripts
       copy_file '_javascript.html.erb', 'app/views/application/_javascript.html.erb'
     end
 
+    def create_shared_css_overrides
+      copy_file '_css_overrides.html.erb', 'app/views/application/_css_overrides.html.erb'
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.